### PR TITLE
App/packages/molecules/card

### PR DESCRIPTION
**Pull request by Mar0cas**

### CODE REVIEW

File: packages/molecules/card/src/component/Utils.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { ImageProps } from '@creat/image';
+import { Props } from './index';
+
+export const setColumnsPerCard = (size: string) => (size === 'big' || !!size ? '4' : '3');
```
Comment (ricardo-valente) [inline packages/molecules/card/src/component/Utils.ts:4]: size should be typified as `Size`

File: packages/molecules/card/src/component/Utils.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+export const setWrapperClass = (columnNumber: number) =>
```
Comment (ricardo-valente) [inline packages/molecules/card/src/component/Utils.ts:6]: same here...

File: packages/molecules/card/src/component/Utils.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    '-gutter-bottom',
+  ].join(' ');
+
+export const setCardClass = ({ size, type, className }: Props) =>
+  ['cards__card', `card--${size}`, `card--${type}`, className].join(' ').trim();
+
+export const getHoverColor = (hover_color: string) =>
+  !!hover_color ? `-hover-${hover_color}` : '';
+
+export const getShadow = (shadow: boolean) => (shadow ? '-shadow' : '');
```
Comment (ricardo-valente) [inline packages/molecules/card/src/component/Utils.ts:21]: ...and here...

File: packages/molecules/card/src/Card.tsx [hunk 4]
```diff
@@ -1,61 +1,60 @@
...
-            ) : null}
+  image,
+}: Props) => (
+  <div className={Utils.setCardClass({ size, type, className })}>
+    <a
+      className={Utils.setCardLinkClass({ color, hover_color, shadow })}
+      href={link}
+      target={link_target}
+    >
+      {!(!!image && type !== 'news') || (
```
Comment (ricardo-valente) [inline packages/molecules/card/src/Card.tsx:32]: even if for some reason we receive an image, if the card type is `news`, the card should not have image

Comment (Mar0cas) [reply packages/molecules/card/src/Card.tsx:32]: ATM we are receiving images on card NEWS, they are being hidden on Twig

File: packages/molecules/card/src/component/Utils.ts (hunk 2 above)
Comment (Mar0cas) [reply packages/molecules/card/src/component/Utils.ts:6]: This was not in use, removed

File: packages/molecules/card/src/index.ts [hunk 5]
```diff
@@ -1,3 +1,5 @@
-import Card from "./Card";
+import Card from './Card';
+import { Color, Defaults, Props, Size, SizeProps, Type, TypeProps, Utils } from './component';
 
+export { Color, Defaults, Props, Props as CardProps, Size, SizeProps, Type, TypeProps, Utils };
 export default Card;
```
[the remark this answers is not in the document]
Comment (Mar0cas) [reply packages/molecules/card/src/index.ts:5]: I wanted to let that out

**Pull request closed without merging**